src/main/java/org/rocstreaming/roctoolkit/RocSender.java

```diff
@@ -159,7 +163,12 @@ public class RocSender extends NativeObject {
     private static long tryOpen(RocContext context, RocSenderConfig config) throws IllegalArgumentException, Exception {
         Check.notNull(context, "context");
         Check.notNull(config, "config");
-        return open(context.getPtr(), config);
+
+        LOGGER.log(Level.FINE, "roc_sender opening, context ptr=" + context.getHexPtr());
+        long ptr = open(context.getPtr(), config);
+        LOGGER.log(Level.FINE, "roc_sender opened, context ptr={0}, ptr={1}",
+                new Object[]{context.getHexPtr(), Long.toHexString(ptr)});
```

Comment:
Consider unifying log messages and using java identifiers, like:

```
starting RocReceiver()
finished RocReceiver()
starting RocReceiver.setMulticastGroup()
finished RocReceiver.setMulticastGroup()
```

feel free to keep them as is too.

Reply:
done